CIPs/0013.md

```diff
@@ -0,0 +1,55 @@
+# CIP 0013: Change BLS hash-to-group (non-SNARK)
+
+- Date: 2020-05-24
+- Author: @kobigurk
```

Comment:
hi @kobigurk  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.